framework/graphics/dx12_util.cpp

```diff
@@ -104,14 +105,40 @@ void TakeScreenshot(std::unique_ptr<graphics::DX12ImageRenderer>& image_renderer
 
                             filename += "_frame_";
                             filename += std::to_string(frame_num);
-                            filename += ".bmp";
-
-                            util::imagewriter::WriteBmpImage(filename,
-                                                             static_cast<unsigned int>(fb_desc.Width),
-                                                             static_cast<unsigned int>(fb_desc.Height),
-                                                             datasize,
-                                                             std::data(captured_image.data),
-                                                             static_cast<unsigned int>(pitch));
+
+                            switch (screenshot_format)
+                            {
+                                default:
```

Comment:
`screenshot_format` can only be unknown if `ScreenshotFormat` is changed. Don't we want to shout about this at compile time so that whoever changes `ScreenshotFormat` comes here and plumbs-in a handler for it before merging their future PR?